crates/rspack_testing/src/test_config.rs

```diff
@@ -584,6 +584,8 @@ impl TestConfig {
     );
     if self.optimization.module_ids == "named" {
       plugins.push(rspack_ids::NamedModuleIdsPlugin::default().boxed());
+    } else if self.optimization.module_ids == "window" {
```

Comment:
This is not ideal. It's better to put the os-specific logic in the same `named` Plugin. It's probably not a good idea to have a special value for Windows.